k8_cortx_cloud/deploy-cortx-cloud.sh

```diff
@@ -733,6 +759,35 @@ function deployCortxConfigMap()
         --from-file="${auto_gen_ha_path}"
 }
 
+# https://stackoverflow.com/a/26665585/508923
+function choose()
+{
+    echo "${1:RANDOM%${#1}:1}" "${RANDOM}";
+}
+
+function choosealpha()
+{
+    choose '0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ'
+}
+
+function pwgen()
+{
+    local first
+    local rest
+    first=$(choosealpha | awk '{printf "%s", $1}')
```

Comment:
What's going on here, is there some component that doesn't allow a non-alpha character as the first?

Reply:
Yes.  This is true for at least one, and I think a couple of the passwords.  I am looking through my notes... I don't see a record of which ones.  I also limited the "special characters" because a couple of the characters didn't translate well through the bash script (not sure if it was b/c of bash/escapes or what... I could have investigated further, but it didn't seem like it would be time well spent).

Reply:
Maybe we should submit bugs against these components. The CSM password policy, for example, is overbearing. Any 8+ password should be accepted.

Reply:
Good point... I'm going with 16 char now.